ajax/libs/3Dmol/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "3Dmol",
+  "filename": "3dmol.min.js",
+  "version": "1.0.6",
+  "description": "Object oriented Javascript molecular visualization library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/3dmol/3Dmol.js.git"
+  },
+  "keywords": [
+    "3Dmol.js",
+    "webGL"
+  ],
+  "author": "David Koes and contributors",
+  "homepage": "http://3dmol.csb.pitt.edu",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/3dmol/3Dmol.js.git",
+    "basePath": "3Dmol",
+    "files": [
+      "3dmol*"
```

Comment:
@kennynaoh I think we need to host the files in `release` folder but I'm not sure. Could you confirm it?